src/omero_cli_render.py

```diff
@@ -696,6 +702,22 @@ def set(self, args):
             if len(active_channels) > 0:
                 img.set_active_channels(active_channels)
 
+            # Set statsInfo min & max
+            for minmax, ch in zip(minmaxlist, img.getChannels(noRE=True)):
+                if minmax[0] is None and minmax[1] is None:
+                    continue
```

Comment:
Would one ever want to null the value that's in the stats info? If so, this continue would prevent that, right?

Reply:
Yes, OK. So if I put `min: null` into the yaml then this line fails `self.min = float(d['min']) if 'min' in d else None`.
I could work around that, but I'd need to set `self.min` to some value that indicates 'unset', since `None` is already used to mean "Don't change anything".
In `JavaScript` we have `undefined` and `null` to choose from, but Python only has `None`!

Reply:
Thinking of the YAML contract from a round-tripping perspective, what it the behavior of `omero render info` for an Image for which the min/max has been nulled in stats info? Does it default to the full pixel type range?

Reply:
@sbesson Yes. E.g. `$ omero render info Image:9621401` for IDR ...`min=0.0,max=65535.0` for all channels.